### ISSUE TITLE
[1.18.2] Quark Azalea and Blossom logs can't be stripped in the cutting board

### ISSUE THREAD

**Issue by Vazkii** [vectorwing/FarmersDelight#470]
smh vector

**Comment by vectorwing** [comment]
smh my head

**Comment by vectorwing** [comment]
I know I replied jokingly before, but I do wonder how to deal properly with this.

As time went on, I mostly removed local recipes for modded wood types, because I couldn't account for some of them changing ID's between versions. Plus, it would be an uphill battle of sorts.

Recently, I learned that Supplementaries has special recipes for its Soap item, which do a registry scan to identify colored items, resulting in the "white" or "uncolored" form automatically. Perhaps I could implement a special recipe for the Cutting Board here, where `x_log` can be stripped into `stripped_x_log` if detected. This could potentially cover hundreds of modded log pairings, so long as I provded some way to deny-list some items.
